src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -17,13 +17,14 @@
 
 import htsjdk.samtools.SAMFileHeader.SortOrder;
```

Comment:
There is a lot of yellow in intellij related to this file.  For example, on line 123 ```validationStringency``` could be final.  Could you fix those up within this file and others.